LayoutTests/fast/mediastream/default-camera-test-expected.txt

```diff
@@ -0,0 +1,4 @@
+
+
+FAIL Check default cameras in case of default device having an unknown facing mode assert_equals: expected "my new camera" but got "Mock video device 1"
```

Comment:
Is this right?

Reply:
Nope, I forgot to rebase the test, but bots seems green, need to check this

Reply:
Actually bots are not happy with the FAIL, I just need to rebase this test.